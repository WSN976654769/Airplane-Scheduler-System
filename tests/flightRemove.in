flight remov
flight remove
location import locations.csv
FLIGHT ADD Monday 5:20 Sydney beijing 20
flight 0 remove
flight 0 remove
exit