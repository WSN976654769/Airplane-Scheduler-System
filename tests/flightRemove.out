User: Invalid Flight ID.

User: Invalid Flight ID.

User: Imported 51 locations.

User: Successfully added Flight 0.

User: Removed Flight 0, Mon 05:20 Sydney --> Beijing, from the flight schedule.

User: Invalid Flight ID.

User: Application closed.
